hw/ip/otbn/dv/smoke/run_smoke.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Runs the OTBN smoke test (builds software, build simulation, runs simulation
+# and checks expected output)
+
+set -o pipefail
+
+if [ ! -f './util/build_consts.sh' ]; then
+  echo "Must be run from OpenTitan repository root"
+  exit 1
+fi
```

Comment:
Tools that care about CWD always feel a bit naff. Let's figure things out properly based on `$BASH_SOURCE`. Something like this maybe? (untested)
```bash
script_dir="$(dirname "$(readlink -e "$BASH_SOURCE[0]")")"
util_dir="$(readlink -e "${script_dir}/../../../../util")" || {
  echo >&2 "Can't find util dir"
  exit 1
}
source "${util_dir}/build_consts.sh"
```